Script/AtomicEditor/editor/Editor.ts

```diff
@@ -123,15 +143,12 @@ class Editor extends Atomic.ScriptObject {
 
     // event handling
     handleExitRequested(data) {
-        Preferences.getInstance().write();
-        EditorUI.shutdown();
-
-    }
-
-    handleEditorEventQuit(data) {
-
-        this.sendEvent("ExitRequested");
-
+      this.subscribeToEvent("AllResourceEditorsClosed", (_) => {
```

Comment:
The event name "AllResourceEditorsClosed" is confusing here, it could happen during normal use when not exiting, you may actually want to do something when this happens.  This is more of a shutdown event?